Cargo.toml

```diff
@@ -19,6 +19,7 @@ netmap = ["netmap_sys"]
 appveyor = []
 travis = []
 with-syntex = ["syntex", "pnet_macros/with-syntex"]
+with-pcap = ["pnet_datalink/with-pcap"]
```

Comment:
Can be `pcap = ["pnet_datalink/pcap"]` if you simply use the feature created by the optional dependency.